src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheck.java

```diff
@@ -73,18 +73,39 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * To configure the check to print warnings if an "@incomplete" tag is found,
- * and not print anything if it is not found:
+ * Example:
```

Comment:
Please add a note that by default, the check does nothing. Thus, there is no default configuration.